shell/platform/windows/client_wrapper/flutter_window_controller.cc

```diff
@@ -0,0 +1,407 @@
+#include "include/flutter/flutter_window_controller.h"
+
+#include "include/flutter/encodable_value.h"
+#include "include/flutter/flutter_win32_window.h"
+#include "include/flutter/standard_method_codec.h"
+
+#include <iomanip>
+#include <sstream>
+
+#include <dwmapi.h>
+
+namespace {
```

Comment:
This `shell/platform/windows/client_wrapper` layer is code that the Flutter tool copies into the end user's project. This code doesn't get compiled into `flutter_windows.dll`. This doesn't seem like the right place to implement the `flutter/windowing` platform channel.

Our other platform channels are registered in the `FlutterWindowsEngine` [here](https://github.com/flutter/engine/blob/33f906426d067812215d46f8cad9f286ab2fdbab/shell/platform/windows/flutter_windows_engine.cc#L211-L226). Should we move the platform channel handling logic down to there?

Reply:
Understood. I'll move the handler to `FlutterWindowsEngine`